src/components/menu/Menu.module.css

```diff
@@ -0,0 +1,30 @@
+
+.image1{
```

Comment:
As with variables, CSS selectors should have meaningful names, so instead of image1, pick a name like `menuImage1` or something like that

Reply:
Apply this to the rest of the file as well :)